tasks/service_rm.rb

```diff
@@ -6,8 +6,8 @@
 require 'puppet'
 
 def service_rm(service)
-  cmd_string = 'docker service rm'
-  cmd_string += " #{service}" unless service.nil?
+  cmd_string = ['docker', 'service', 'rm']
+  cmd_string += [" #{service}"] unless service.nil?
```

Comment:
We should try to avoid using += here.

I believe [Array#concat](https://ruby-doc.org/core-2.1.1/Array.html#method-i-concat) would be preferable in this instance.

Also lets rename the variable to cmd rather than cmd_string.

Reply:
See above.

Reply:
In this case you have changed from concatenating a string to an array so we should generally lean towards the more idiomatic approach.